blkn/src/Input/ErrorLine.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import * as React from "react";
+import Typography from "./../Typography";
+
+type Props = {
+  error?: string,
+};
+
+const ErrorLine = (props: Props) =>
+  props.error ? (
```

Comment:
if we're rendering an `Error`, it is imperative it already exists. also, don't do useless `div` wrapping.

either make the `error` prop mandatory, or name the component `MaybeError`

also, use `{error && </>}` instead of ternary

Reply:
`error && </>` might result in `undefined` and that will produce an error ("Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.")

Reply:
k that makes sense. consider the first two comments, though

Reply:
I'll rename it to MaybeError.